CRM/Upgrade/Incremental/php/FiveTwo.php

```diff
@@ -61,6 +61,41 @@ public function setPostUpgradeMessage(&$postUpgradeMessage, $rev) {
     // }
   }
 
+  /**
+   * Upgrade function for version 5.3.alpha1. This upgrader adds the default assignee
+   * option values that can be selected when creating or editing a new
+   * workflow's activity.
+   *
+   * @param string $rev
+   */
+  public function upgrade_5_3_alpha1($rev) {
```

Comment:
Should not it be 5_2_alpha1?

Reply:
It should be 5.3.alpha1 because that's the version in master.

Reply:
I just needed to move it to another file.